rapids-cmake/cpm/patches/command_template.cmake.in

```diff
@@ -62,13 +63,16 @@ function(rapids_cpm_run_git_patch file issue)
   endif()
 
   if(result EQUAL 0)
-    list(APPEND msg_state "rapids-cmake [@package_name@]: applied ${ext} ${file_name} to fix issue: '${issue}'")
+    list(APPEND msg_state "rapids-cmake [@package_name@]: applied ${ext} ${file_name} to fix issue: '${issue}'\n")
+  elseif(require)
+    list(APPEND error_state "rapids-cmake [@package_name@]: failed to apply ${ext} ${file_name} \n")
+    list(APPEND error_state "rapids-cmake [@package_name@]: git ${ext} output: ${repo_error_info} \n")
   else()
-    list(APPEND msg_state "rapids-cmake [@package_name@]: failed to apply ${ext} ${file_name}")
-    list(APPEND msg_state "rapids-cmake [@package_name@]: git ${ext} output: ${repo_error_info}")
+    list(APPEND msg_state "rapids-cmake [@package_name@]: failed to apply ${ext} ${file_name}\n")
+    list(APPEND msg_state "rapids-cmake [@package_name@]: git ${ext} output: ${repo_error_info}\n")
```

Comment:
To avoid repeating the message, you could store "error_state" or "msg_state" to a new var ${output_state} and then append to that.

Reply:
Updated.